apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -198,13 +199,16 @@ export default function Default() {
   const listAppsV2 = useFeature("listAppsV2minor1");
   const analyticsFF = useFeature("lldAnalyticsOptInPrompt");
   const hasSeenAnalyticsOptInPrompt = useSelector(hasSeenAnalyticsOptInPromptSelector);
+  const isLocked = useSelector(isLockedSelector);
   const dispatch = useDispatch();
 
   useEffect(() => {
-    if (!analyticsFF?.enabled || hasSeenAnalyticsOptInPrompt) return;
-    dispatch(setShareAnalytics(false));
-    dispatch(setSharePersonalizedRecommendations(false));
-  });
+    if (!isLocked && analyticsFF?.enabled && !hasSeenAnalyticsOptInPrompt) {
```

Comment:
why are we changing the logic? previously it was an or case?

Reply:
It does the same as before because now we execute the logic instead of returning when the if is true

Reply:
Yes it was an or case but we want to trigger the theses dispatches only if those 3 conditions are met: 
- App unlocked
- feature flag is enabled
- the prompt has not been seen by the user

The OR case was a return if the user didn't see the prompt or if the FF is disabled 
So we have the same things here as before

Reply:
yikes